Assets/HotmaxEditorVR/rotationControl.cs

```diff
@@ -0,0 +1,215 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class rotationControl : MonoBehaviour
+{
+    //--local refs
+    private bool _rotationGizmoIsSelected = stateManager.rotationGizmoIsSelected;
+    private stateManager.rotationModes _rotationMode = stateManager.rotationMode;
+
+    //--vars used during rotation
+    private bool initialHit = false;
+    private Transform initialParent;
+    private Vector3 targetPostition;
```

Comment:
typo

Reply:
turns out i wasn't using that var anywhere, deleted